Sources/DefaultsMacrosDeclarations/ObservableDefaultMacro.swift

```diff
@@ -0,0 +1,117 @@
+import SwiftCompilerPlugin
+import SwiftDiagnostics
+import SwiftSyntax
+import SwiftSyntaxBuilder
+import SwiftSyntaxMacros
+
+public struct ObservableDefaultMacro: AccessorMacro {
+	public static func expansion(
+		of node: AttributeSyntax,
+		providingAccessorsOf declaration: some DeclSyntaxProtocol,
+		in context: some MacroExpansionContext
+	) throws -> [AccessorDeclSyntax] {
```

Comment:
```suggestion
	) throws(ObservableDefaultMacroError) -> [AccessorDeclSyntax] {
```

And then you can remove `ObservableDefaultMacroError` from the throw statements.

Reply:
Ah! typed throws are here and I forgot about them. Neat.